.github/workflows/generate.yaml

```diff
@@ -0,0 +1,52 @@
+name: Generate and Test on OAS change
+
+on:
+  pull_request:
+    paths:
+      - 'openapi.json'
+      - 'api.yaml'
+    types:
+      - opened
+
+jobs:
+  generate_and_test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+
+    - name: Install dependencies
+      run: npm install -g konfig-cli
+
+    - name: Regenerate SDKs
+      run: konfig generate
+
+    - name: Create changeset file
+      run: konfig changeset -a -m "Regenerate SDKs"
+
+    - name: Commit changes
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
```

Comment:
is this necessary

Reply:
if we don't do that then "the commit will be attributed to the default Git user and email associated with the runner environment where the workflow is executed. The default Git user and email are typically set globally on the runner or repository level. If they haven't been explicitly configured, Git will use the system's default values. "

so probably will say you committed it Im guessing

Reply:
thats fine lets just use the system default for now.

Reply:
actually looks like this is required

```
Run git commit -am "Regenerate SDKs"
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'runner@fv-az[4](https://github.com/konfig-dev/acme-sdks/actions/runs/5944522040/job/16121964907?pr=64#step:7:5)49-83[7](https://github.com/konfig-dev/acme-sdks/actions/runs/5944522040/job/16121964907?pr=64#step:7:8).(none)')
Error: Process completed with exit code 12[8](https://github.com/konfig-dev/acme-sdks/actions/runs/5944522040/job/16121964907?pr=64#step:7:9).
```